src/libs/OptionsListUtils.ts

```diff
@@ -1428,6 +1428,7 @@ function getValidOptions(
     // Gather shared configs:
     const loginsToExclude: Record<string, boolean> = {
         [CONST.EMAIL.NOTIFICATIONS]: true,
+        [CONST.EMAIL.MANAGER_MCTEST]: true,
         ...excludeLogins,
```

Comment:
```suggestion
        ...excludeLogins,
         [CONST.EMAIL.MANAGER_MCTEST]: true,
```
I think this should fix the problem